javascript/index.js

```diff
@@ -0,0 +1,61 @@
+let books = JSON.parse(localStorage.getItem('books'));
+localStorage.setItem('books', JSON.stringify(books));
+let storedBooks = JSON.parse(localStorage.getItem('books'));
+
+function removeBook() {
+  storedBooks = JSON.parse(localStorage.getItem('books'));
+  for (let index = 0; index < storedBooks.length; index += 1) {
+    if ((this.parentElement.children[0].innerText === storedBooks[index].title)
+      && (this.parentElement.children[1].innerText === storedBooks[index].author)) {
+      storedBooks.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(storedBooks));
+      storedBooks = JSON.parse(localStorage.getItem('books'));
+      window.location.reload();
+    }
+  }
+}
```

Comment:
**Changes Required**
Good effort on implementing the remove function, however;
- [ ] Currently your book object doesn't have a unique attribute for instance `id`. When a add more than one book with the same title, the `remove function` doesn't work or it doesn't remove the exact book object. Let us try and fix this bug.

Reply:
Thanks for the observation. We decided to tackle the root of the problem. we ensured that the same books is not repeated in the catalog.